apps/prairielearn/src/pages/administratorSettings/administratorSettings.ts

```diff
@@ -1,10 +1,13 @@
 import * as express from 'express';
 import asyncHandler from 'express-async-handler';
+import OpenAI from 'openai';
 
 import { cache } from '@prairielearn/cache';
 import * as error from '@prairielearn/error';
 
+import { syncContextDocuments } from '../../ee/lib/contextEmbeddings.js';
```

Comment:
is there an issue with linking enterprise-edition code within a full edition page?

Reply:
Typically we'd guard this all with [`isEnterprise()`](https://github.com/PrairieLearn/PrairieLearn/blob/ce72108f23b70116e1af5afa48c598d66e811f47/apps/prairielearn/src/lib/license.js#L13) calls. In this case, you can [`await import(...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import) the context embeddings file in the handler.

Reply:
Done in 99f9ce0e0871e9b04042c1180fd183dcf2f45434.